test/e2e/memcached_test.go

```diff
@@ -30,6 +30,8 @@ func TestMemcached(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error: %v\nCommand Output: %s\n", err, string(cmdOut))
 	}
+	defer os.RemoveAll(os.Getenv("GOPATH") + "/src/github.com/example-inc/memcached-operator")
```

Comment:
It seems to me that `os.Getenv("GOPATH") ` is called at multiple places. we can avoid that by first saving the value to a variable such as 

```go
gopath, ok :=  os.LookupEnv("GOPATH")
if !ok {
  t.Fatal(...)
}
```